helpers/template.py

```diff
@@ -215,7 +215,9 @@ def _get_value(property_, true_value="", false_value="#",
             "REDIS_MAIN_PORT": config.get("redis_main_port", "6739"),
             "REDIS_CACHE_PORT": config.get("redis_cache_port", "6380"),
             "USE_BACKUP": "" if config.get("use_backup") == Config.TRUE else "#",
-            "USE_WAL_E_BACKUP": "" if config.get("use_wal_e") == Config.TRUE else "#",
+            "USE_WAL_E_BACKUP": _get_value("use_wal_e", true_value="",
```

Comment:
default values for `_get_value()` are:
- `true_value=""`
- `false_value="#"`,
- `comparison_value=Config.TRUE`

For this particular case, you could just call `_get_value('use_wal_e')`